scripts/startup.sh

```diff
@@ -365,7 +365,7 @@ case $? in
     *) set_option FORMATEFI "no" ;;
     esac
 
-  elif [[ $(sudo sudo fdisk -l | grep -i '^Disklabel type') = "Disklabel type: gpt" ]]; then # Checking for GPT Disk Label on a Legacy BIOS (non UEFI) System
+  elif [[ $(fdisk -l | grep -i 'Disklabel type') = "Disklabel type: gpt" ]]; then # Checking for GPT Disk Label on a Legacy BIOS (non UEFI) System
```

Comment:
This if statement is true even if only one of many disks in the device is using GPT. I am working on a fix to make it check the drive that contains the partition selected by the user only.

Reply:
https://github.com/ChrisTitusTech/ArchTitus/pull/293/commits/4eeaa78c075093b7a8c78c79c71fb4431d10d79a solved this.

### Changes
* scripts/startup.sh now asks the user which drive to select a partition from before asking to select the partition itself. The selected drive is then checked for GPT in the new version of the above if statement.